### PR TITLE
test(accessibility): update for urlList creation

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- change for `accessibility` urlList 
 
### Current behaviour

- we had a list where we were excluding components

### Checklist

- [ ] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Related docs have been updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

File: cypress/support/accessibility/a11y-utils.js [hunk 1]
```diff
@@ -41,78 +41,18 @@ export default (from, end) => {
...
-      !prepareUrl[0].startsWith("numeral-date") &&
-      !prepareUrl[0].startsWith("global-header") &&
-      !prepareUrl[0].startsWith("grid") &&
-      !prepareUrl[0].startsWith("image") &&
-      !prepareUrl[0].startsWith("navigation-bar") &&
-      !prepareUrl[0].startsWith("popover-container") &&
-      !prepareUrl[0].startsWith("menu") &&
+      (prepareUrl[0].startsWith("flat-table") ||
+        prepareUrl[0].startsWith("hr") ||
+        prepareUrl[0].startsWith("inlile-inputs") ||
```
Comment (divyajindel) [inline cypress/support/accessibility/a11y-utils.js:46]: @DlgSHi Could you please remove "inline-inputs" as well from here as this component has been merged already.

**Review by stephenogorman — APPROVED** [review]

File: cypress/support/accessibility/a11y-utils.js [hunk 2]
```diff
@@ -41,78 +41,18 @@ export default (from, end) => {
...
+        prepareUrl[0].startsWith("radiobutton") ||
+        prepareUrl[0].startsWith("select") ||
+        prepareUrl[0].startsWith("showeditpod") ||
```
Comment (DipperTheDan) [inline cypress/support/accessibility/a11y-utils.js:49]: We're deprecating `ShowEditPod`. Is it worth removing this now or when it's deleted from the codebase?

Comment (sianford) [reply cypress/support/accessibility/a11y-utils.js:49]: We'll leave it in til it's removed from the codebase.

Comment (DipperTheDan) [reply cypress/support/accessibility/a11y-utils.js:49]: Seeing as @sianford has answered my question. I'll resolve this comment ✅

**Review by DipperTheDan — COMMENTED** [review]
Just resolve the merge conflict and I'll approve this 👍🏻

**Pull request closed without merging**